proxy/ssl/key.pem

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
can you add the generation script here so we know that these are just self-signed?